contrib/playlib/worker/2.8/src/RouteCompilerWorker.scala

```diff
@@ -0,0 +1,79 @@
+package mill
+package playlib
+package worker
+
+import java.io.File
+
+import mill.playlib.api.{RouteCompilerType, RouteCompilerWorkerApi}
+import play.routes.compiler
+import play.routes.compiler.RoutesCompiler.RoutesCompilerTask
+import play.routes.compiler.{InjectedRoutesGenerator, RoutesCompilationError, RoutesCompiler, RoutesGenerator}
+
+
+private[playlib] class RouteCompilerWorker extends RouteCompilerWorkerApi {
+
+  override def compile(files: Array[java.io.File],
+                       additionalImports: Array[String],
+                       forwardsRouter: Boolean,
+                       reverseRouter: Boolean,
+                       namespaceReverseRouter: Boolean,
+                       generatorType: RouteCompilerType,
+                       dest: java.io.File): String = {
+    generatorType match {
+      case RouteCompilerType.InjectedGenerator =>
+        val result = compileWithPlay(files.map(os.Path(_)), additionalImports, forwardsRouter, reverseRouter,
+          namespaceReverseRouter, os.Path(dest), InjectedRoutesGenerator)
+        asMillResult(result)
+      case RouteCompilerType.StaticGenerator =>
```

Comment:
Does just returning a string here means, mill will not detect any error but continue it's work?